src/js/view/controls/rightclick.js

```diff
@@ -91,6 +91,7 @@ export default class RightClick {
 
         addClass(this.playerElement, 'jw-flag-rightclick-open');
         addClass(this.el, 'jw-open');
+        this.el.focus();
```

Comment:
I'm worried about the scrolling behavior we saw before. Would `this.playerElement.blur()` work just as well? (we could also remove the template changes below)

Reply:
I'll try that out!

Reply:
That wouldn't fix the problem with focusing on the settings menu when we rightclick on those buttons